.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "src/shared"]
 	path = src/shared
-	url = git@github.com:Chexxo/CommonTypes.git
+	url = https://github.com/Chexxo/CommonTypes.git
```

Comment:
Does this have to be commited?

Reply:
We did it like this before the change from CommonTypes to shared. Wouldn't that also make the step to fix up git urls obsolete in the CI?

Reply:
No since i use ssh and that then may break the CI. But we can leave it at that for the moment.